libs/blocks/preflight/preflight.css

```diff
@@ -686,6 +686,7 @@ picture:has(.picture-meta) {
   line-height: 1.2;
   z-index: 9;
   opacity: 0.7;
+  height: auto !important;
```

Comment:
As wise man @overmyheadandbody usually states, let's avoid the `!important` - that should be the absolute last thing we consider if all fails

Reply:
If we can get away slightly refactoring what currently exists, that'd be preferred